docs/static/plugin-manager.asciidoc

```diff
@@ -112,6 +112,22 @@ bin/logstash-plugin update logstash-input-github <2>
 <1> updates all installed plugins
 <2> updates only the plugin you specify
 
+[discrete]
+[[updating-plugins-major-versions]]
+==== Major-version plugin updates
+
+By default, the plugin manager will only update plugins for which newer _minor_ or _patch_ versions exist, to avoid the introduction of breaking changes.
```

Comment:
```suggestion
To avoid introducing breaking changes, the plugin manager updates plugins for which newer minor or patch versions exist by default. 
```